src/plugins/plot/chart/MctChart.vue

```diff
@@ -725,7 +725,9 @@ export default {
       });
     },
     showLabels(seriesKey) {
-      return this.showLimitLineLabels.seriesKey && this.showLimitLineLabels.seriesKey === seriesKey;
+      return (
+        this.showLimitLineLabels?.seriesKey && this.showLimitLineLabels.seriesKey === seriesKey
```

Comment:
Think you can simplify this: `return this.showLimitLineLabels?.seriesKey  === seriesKey`